Automattic/jetpack
# WP.com settings menu: add newsletter & podcasting sections

Goes together with 
- Newsletters https://github.com/Automattic/wp-calypso/pull/79986
- Podcasting https://github.com/Automattic/wp-calypso/pull/79996

## Proposed changes:
* Add a new submenus under "Settings" for WP.com for newsletters and podcasting.
* No changes to Jetpack sites.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
- Apply the change at WP.com sandbox as [per instructions below](https://github.com/Automattic/jetpack/pull/32146#issuecomment-1658220786)
- Enable filters in your sandbox:
   ```php
   add_filter( 'calypso_use_newsletter_settings', '__return_true' );
   ```
   ```php
   add_filter( 'calypso_use_podcasting_settings', '__return_true' );
   ```
- See under "Settings" in Calypso

<img width="288" alt="Screenshot 2023-07-31 at 15 21 07" src="https://github.com/Automattic/jetpack/assets/87168/71ed334c-8f12-487c-891d-01aeae8362c1">
